tacs/constitutive.pyx

```diff
@@ -465,6 +465,62 @@ cdef class PlaneStressConstitutive(Constitutive):
             self.ptr = NULL
             self.cptr = NULL
 
+cdef class PhaseChangeMaterialConstitutive(Constitutive):
+    """
+    This is the base class for the plane stress constitutive objects.
```

Comment:
Can you update the doctoring here? It still describes this as a plane stress constitutive